cmd/keel/main.go

```diff
@@ -46,6 +46,7 @@ import (
 	_ "github.com/keel-hq/keel/extension/notification/mail"
 	_ "github.com/keel-hq/keel/extension/notification/mattermost"
 	_ "github.com/keel-hq/keel/extension/notification/slack"
+	_ "github.com/keel-hq/keel/extension/notification/teams"
```

Comment:
The purpose of this is unclear to me. There was no reference to these extensions in main.go.